tsconfig.json

```diff
@@ -10,6 +10,7 @@
     },
     "strict": true,
     "strictPropertyInitialization": false,
+    "suppressImplicitAnyIndexErrors": true,
```

Comment:
What's the reason behind this configuration?

Reply:
@eranshmil forgot to remove :wink: